### PR TITLE
feat(theme): add colors + text styles

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
previously, we had no themes, causing engineers to reimplement stuff across 

## Solution
- structure files in a broadly similar way to our FE
- add themes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,6 @@
 .bundle
 .sass-cache
 _site/
-Gemfile.lock
```
Comment (harishv7) [inline .gitignore:5]: why was this even in gitignore so far?

File: _sass/theme/_colors.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+$canvas-base: #ffffff;
+$content-base: #000000;
+$utility-theme-color: $primary-color;
```
Comment (harishv7) [inline _sass/theme/_colors.scss:3]: Where does this come from?

File: assets/css/blueprint.css [hunk 3]
```diff
@@ -1,12 +1,5 @@
 @charset "UTF-8";
 
-@import "components/hero.css";
-@import "theme/padding.css";
-@import "theme/text-styles.css";
-@import "theme/margin.css";
-@import "theme/layout.css";
-@import "theme/background.css";
```
Comment (harishv7) [inline assets/css/blueprint.css:8]: are there lines in blueprint which depend on these imports?

File: _sass/theme/_colors.scss (hunk 2 above)
Comment (seaerchin) [reply _sass/theme/_colors.scss:3]: this comes from our `styles.scss` - we have to use `import` sadly due to the version of `sass-embedded` not supporting `use`. upgrading it causes errors on m1 mac, which i'm abit scared to do (fixed on local) given the age of the script

File: .gitignore (hunk 1 above)
Comment (seaerchin) [reply .gitignore:5]: very good question; last change was 5 years ago so i suspect teh answer has been lost to time

File: assets/css/blueprint.css (hunk 3 above)
Comment (seaerchin) [reply assets/css/blueprint.css:8]: no - these were introduced by me for theming but zj found a way to use scss so i'm swapping over (in this PR)

File: .gitignore (hunk 1 above)
Comment (harishv7) [reply .gitignore:5]: oops

**Review by harishv7 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (kishore03109) [reply .gitignore:5]: wait no this should be here right? this file is created when we build the jekyll app locally no?

Comment (seaerchin) [reply .gitignore:5]: not sure what you mean here; this is the equivalent of `package.lock`

Comment (harishv7) [reply .gitignore:5]: yea is a lockfile, shld be commited

**Pull request merged**